### PR TITLE
Admin Clean Up on ecip-1049.md

### PR DESCRIPTION
**Pull request by gitr0n1n**
+ move to Draft Status (Expired Last Call Review on Oct 9, 2020) due to material redraft of proposal.
+ Update Discussion Thread to new Redraft thread
+ Add Related Discussions Section for archived threads

### CODE REVIEW

**Comment by q9f** [comment]
I think we can continue working on that document, so I would rather send it back to draft status rather than rejecting it.

**Comment by gitr0n1n** [comment]
> I think we can continue working on that document, so I would rather send it back to draft status rather than rejecting it.

Can that direct movement to `Draft` be made? source: the ECIP diagram.

I see that `Last Call` status can only be moved to `Accepted` or `Rejected`.

`Last Call - An ECIP that is done with its initial iteration and ready for review by a wider audience.`
https://github.com/ethereumclassic/ECIPs/raw/master/assets/ecip-1000/process.png

Pedantic, but thats how the Editors have chosen to follow the ECIP process for all ECIPs.

So move ECIP-1049 to Rejected. Alex resubmits the materially changed `reDraft` proposal to Draft status. Will that require a new ECIP # (like all the Aztlan proposals)?  @q9f

**Comment by kizoojun01** [comment]
kizoojun@kizoojunkoi.org

**Review by kizoojun01 — CHANGES_REQUESTED** [review]
kizoojun@kizoojunki.org

**Comment by gitr0n1n** [comment]
> I think we can continue working on that document, so I would rather send it back to draft status rather than rejecting it.

@q9f can we get this ECIP moved back to the status it is currently in: `Draft`.

I believe we might needed to have it `Withdrawn` then resubmitted as `Draft`. The issue is that this proposal has past its current review period and now the authors are re-drafting the proposal. It's almost been in this limbo state for 60 days. I think its time to clean this up. I'd do it myself if I was an editor, but those PRs are also sittingon the shelf.

Thanks in advance for the ECIP housekeeping.

**Comment by q9f** [comment]
I would supporting moving it back to Draft for the time being

**Comment by bobsummerwill** [comment]
That would make sense to me too, @q9f.

The flow chart in ECIP-1000 does not show a path from Last Call back to Draft, but I don't think we gain anything by forcing the creation of a new, identical ECIP just for the sake of rigid adherence.

**Comment by TokenHash** [comment]
@gitr0n1n @q9f @bobsummerwill ECIPs can perfectly be moved back to draft. The ECIP process does not specifically say that it can't and the drawing is just a diagram. I drew it like that cause I was copying the Bitcoin one (see below) and for some reason put arrows instead of just lines. I guess I unconsciously put arrows as I was thinking in the forward process, and not about special cases where ECIPs needed to go backward.

bip-process

The ECIP process has predecessors in the BIP and EIP processes. EIP says clearly:

```Last Call - This is the final review window for an EIP before moving to FINAL. An EIP editor will assign Last Call status and set a review end date (review-period-end), typically 14 days later.

If this period results in necessary normative changes it will revert the EIP to REVIEW.

Final - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.

Stagnant - Any EIP in DRAFT or REVIEW if inactive for a period of 6 months or greater is moved to STAGNANT. An EIP may be resurrected from this state by Authors or EIP Editors through moving it back to DRAFT.```

BIP say clearly:

```BIPs should be changed from Draft or Proposed status, to Rejected status, upon request by any person, if they have not made progress in three years. Such a BIP may be changed to Draft status if the champion provides revisions that meaningfully address public criticism of the proposal, or to Proposed status if it meets the criteria required as described in the previous paragraph.```

I tried to change the diagram when Wei Tang was harassing the ETC ecosystem arguing the same thing about no backward status changes. The supposed impossibility of moving ECIPs backward to draft was always false, because it is obvious that when an ECIP has a catastrophic problem, no matter what is the status, they have to be moved back to draft to correct issues (as prescribed by the BIP and EIP). 

This has happened in ETH, but I don't remember what case it was.

In conclusion, ECIPs can be moved to draft (I co-wrote ECIP-1000). I will enter the corrected diagram to the assets folder so it is corrected and does not create these confusions again.

Wei was controlling the ECIP-1000 repo at the time of the argument and constantly sabotaging other ECIPs and commentary as he had editor, writing, and author access and authority. That is why I could never change the diagram.

**Comment by TokenHash** [comment]
@gitr0n1n @q9f @bobsummerwill  

This is the correct diagram for ECIP-1000:

process

...but I think I messed up when creating the PR from my fork (I haven't used GitHub for a long time  ;)

could one of you add the process.png image to the assets folder pls?

File: _specs/ecip-1049.md [hunk 1]
```diff
@@ -2,11 +2,10 @@
 lang: en
 ecip: 1049
 title: Change the ETC Proof of Work Algorithm to Keccak256
-status: Last Call
-review-period-end: 2020-10-09
+status: Rejected
```
Comment (TokenHash) [inline _specs/ecip-1049.md:5]: Please change this to `Draft`.

Comment (gitr0n1n) [reply _specs/ecip-1049.md]: Resolved.

**Comment by gitr0n1n** [comment]
> @gitr0n1n @q9f @bobsummerwill
> 
> This is the correct diagram for ECIP-1000:
> 
> process
> 
> ...but I think I messed up when creating the PR from my fork (I haven't used GitHub for a long time ;)
> 
> could one of you add the process.png image to the assets folder pls?
> 
> Note that the diagram above is the new correct one, the previous one was not even updated to all the possible statuses that ECIPs may have.

Thanks for clarifying that diagram.

**Comment by TokenHash** [comment]
@gitr0n1n I just re-learned how to use GitHub....lol....so I entered a PR to update the diagram.

**Review by TokenHash — APPROVED** [review]

**Comment by bobsummerwill** [comment]
Wei must have polluted my brain with his rigidity about the process, @TokenHash!

**Review by q9f — APPROVED** [review]

**Comment by q9f** [comment]
Thanks!

**Review by Miami87 — APPROVED** [review]
